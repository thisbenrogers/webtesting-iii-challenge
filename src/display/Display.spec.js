import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';

import Display from './Display';

describe("<Display />", () => {

    it('renders <Display /> correctly', () => {
      const tree = renderer.create(<Display />).toJSON();
      expect(tree).toMatchSnapshot();
    });

    
});

describe("component", () => {
    it("Should display 'Unlocked', 'Open', and 'Close Gate'", () => {
        const { queryByText } = render(<Display />);
        expect(queryByText("Unlocked")).toBeTruthy();
        expect(queryByText("Open")).toBeTruthy();
    });
});
