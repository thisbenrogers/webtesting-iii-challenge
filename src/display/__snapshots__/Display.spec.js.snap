// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Display /> renders <Display /> correctly 1`] = `
<div
  className="display panel"
>
  <div
    className="led green-led"
  >
    Unlocked
  </div>
  <div
    className="led green-led"
  >
    Open
  </div>
</div>
`;
